### PR TITLE
faq/tidb: fix mis-linked file in FAQ

### PR DESCRIPTION
**Pull request by dcalvin**
### What is changed, added or deleted? (required)

This PR fixed an incorrect link that pointed to a wrong place and added back the right file removed in refactoring. 

### What is the related PR or file link(s)? (required) 

https://github.com/pingcap/docs-cn/pull/1458

### Which version does your change affect? (required)

V3.0

### CODE REVIEW

**Comment by lilin90** [comment]
@morgo FYI. Some links point to the content in the deleted file, so we add it back and will remove it when new files are ready.

**Comment by lilin90** [comment]
Do we need to add an alias for this file? @dcalvin

**Comment by dcalvin** [comment]
> Do we need to add an alias for this file? @dcalvin

We can add one in case any user has bookmarked this page.

File: dev/reference/configuration/tidb-server/server-command-option.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+---
+title: The TiDB Command Options
+summary: Learn about TiDB command options and configuration files.
+category: user guide
```
Comment (dcalvin) [inline dev/reference/configuration/tidb-server/server-command-option.md]:
```suggestion
category: user guide
aliases: ['/docs/sql/server-command-option/']
```

**Comment by morgo** [comment]
LGTM. If we need the file, can we add it to the TOC?

**Comment by dcalvin** [comment]
> LGTM. If we need the file, can we add it to the TOC?

I think it makes sense, as it contains options that could not otherwise be found. We can add it to TiDB server configs

**Pull request merged**